src/qt_gui/background_music_player.cpp

```diff
@@ -0,0 +1,36 @@
+// SPDX-FileCopyrightText: Copyright 2024 shadPS4 Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "background_music_player.h"
+
+BackgroundMusicPlayer::BackgroundMusicPlayer(QObject* parent) : QObject(parent) {
+    m_mediaPlayer = new QMediaPlayer(this);
+    m_audioOutput = new QAudioOutput(this);
+    m_mediaPlayer->setAudioOutput(m_audioOutput);
+}
+
+BackgroundMusicPlayer::~BackgroundMusicPlayer() {
+    delete m_mediaPlayer;
+    delete m_audioOutput;
```

Comment:
AFAIK you should not explicitly delete objects that are part of Qt's object model (created with `parent` object parameter) as they will be deleted when `parent` is destroyed.